gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc

```diff
@@ -0,0 +1,286 @@
+= Add Flex Gateway Replicas to a Namespace
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+For high availability (HA) environments, horizontal scaling distributes load across multiple Flex Gateway instances. By default, the Helm chart for Flex Gateway is configured with one replica. To release more Flex Gateway replicas into a namespace, provide one of the following configurations:
+
+* <<replica-count, Set a replica count>>: Set a fixed number of Flex Gateway replicas to create.
+* <<autoscale, Configure autoscaling>>: Generate replicas automatically based on CPU, memory, and other settings. 
+
+//how to view helm chart defaults:
+include::partial$reference-k8-general.adoc[tags=flex-k8-helm-chart-info]
+
+== Before You Begin
+
+Ensure that the following prerequisites are in place:
+
+* A Flex Gateway deployment or a registered Flex Gateway instance (gateway)
++
+For gateway registration and deployment processes, see xref:flex-gateway-set-up.adoc[] or xref:flex-gateway-k8-getting-started.adoc[]. 
+
+* A Kubernetes cluster for your Flex Gateway deployment
+
+[[replica-count]]
+== Set a Replica Count
+
+Configure and verify the number of replicas that you require. The steps to follow depend on whether a Helm chart is installed. 
```

Comment:
require -> need?

I read "The steps to follow" as equal to "The following steps" not "The steps to take".

Reply:
One idea...
. Depending on your configuration, run the following command:
** If a Helm chart is installed:
** If a Helm chart is not installed:

Reply:
I changed it to "The steps to take," which sounds better to me. Re the alternative, if they are installing for the first time, they don't have a config yet, so can't really say "Depending on your configuration."

Reply:
The reason I suggest the bullet list approach is that steps imply linear (do this, then this) and bullets imply options. (At least, to me.)

Reply:
Yeah, I started that way, but then I decided that the space between the bullets made it less easy to use. There were some other issues that I don't remember off the top of my head right now. But I'm usually thinking about stuff like this when I write docs.